tools/experiments/experiments.html

```diff
@@ -212,6 +212,11 @@ <h2>Release channels</h2>
   </section>
 
   <section class="content">
+      <p>
+        <em>Note:</em> these controls set a cookie on the domain name <code>cdn.ampproject.org</code>. If your browser is
+        set to block all third-party cookies, they will not function. Make sure you disable third-party cookie blocking,
```

Comment:
For readability, I suggest rephrasing `disable third-party cookie blocking` (double-negative) to `Make sure third-party cookies are enabled, or whitelist...`.

Reply:
Done with tiny change. PTAL